src/stdatamodels/jwst/datamodels/schemas/core.schema.yaml

```
@@ -927,6 +927,14 @@ properties:
             type: boolean
             fits_keyword: SELFREF
             blend_table: True
+          primary_cosmic_rays:
+            title: Rate of Cosmic Rays per thousand pixels per second
```

```suggestion
            title: Cosmic Ray rate per thousand pixels per second
```
Original title string is too long to fit within FITS rec without being truncated.